Neos.Flow/Classes/Configuration/ConfigurationManager.php

```diff
@@ -482,7 +482,7 @@ public function refreshConfiguration(): void
     }
 
     /**
-     * Replaces variables (in the format %CONSTANT% or %env:ENVIRONMENT_VARIABLE%)
+     * Replaces variables (in the format %CONSTANT%, %env:ENVIRONMENT_VARIABLE% or %file:%PATH_TO_FILE%)
```

Comment:
documentation contains an extra `%` before `PATH_TO_FILE`.
I'd also suggest to write the example as `or %file:/path/to/file%`.